### PR TITLE
feat(vector): Bump Vector version to 0.28.0

### PR DESCRIPTION
**Pull request by jszwedko**
Signed-off-by: Jesse Szwedko <jesse.szwedko@datadoghq.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
...
+### Vector
+
+#### Bug Fixes
+
+- Bump Vector to 0.27.1 (#284) ([bca819f](https://github.com/vectordotdev/helm-charts/commit/bca819f25e4ee3733b9f20d8ddfc7bf35ae5db1f))
+
+#### Features
+
+- Add annotations for deployment, daemonset, statefulset (#280) ([bce0b77](https://github.com/vectordotdev/helm-charts/commit/bce0b77319df7bac0b0ba0743438f0e0ce338dda))
+
```
Comment (spencergilbert) [inline CHANGELOG.md:17]: This got double added? It's under 0.19.2 as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.20.0] - 2023-02-27
+
+### Vector
+
+#### Bug Fixes
+
+- Bump Vector to 0.27.1 (#284) ([bca819f](https://github.com/vectordotdev/helm-charts/commit/bca819f25e4ee3733b9f20d8ddfc7bf35ae5db1f))
```
Comment (spencergilbert) [inline CHANGELOG.md:12]: I'm only noticing this now, but shouldn't 0.19.2 have this note and this PR have the 0.28 note?

File: CHANGELOG.md (hunk 1 above)
Comment (jszwedko) [reply CHANGELOG.md:17]: Ah, hmm, I think what happened is I needed a separate commit before generating the CHANGELOG for 0.19.2. Let me see if I can fix that.

**Comment by jszwedko** [comment]
@spencergilbert thanks for catching. I think I was able to rectify the CHANGELOG in this PR 🤞

**Review by spencergilbert — APPROVED** [review]

**Comment by spencergilbert** [comment]
Unclear why the tests failed there, seems to be totally in the realm of the 3rd party action - so I just retried.

**Pull request merged**